### DISCUSSION TITLE
Problem wiith driven entity  Versatile Thermostat

### DISCUSSION THREAD

**Discussion by pgiulio** [jmcollin78/versatile_thermostat#370] [Q&A]
I want to eliminate soft on and off. I want the heating activation to always remain active until the set point is reached. Now the output continues to turn on and off every 20 minutes.

![image](https://github.com/jmcollin78/versatile_thermostat/assets/75814425/00a0d3cb-8735-4d74-a757-c2a82759ff78)

**Comment by jmcollin78** [comment]
Hello @pgiulio ,

With VTherm as is, this is not possible. But why do you want to do that ? What is your case ? VTherm aims to regulated and avoid over heating. And what you want to do is totally on the contrary of this goals.

If the temperature reached is under the target, we can augment the coef_int and/or coef_ext.

**Comment by MBRetrofit** [comment]
At 100% request, it will remain active without "soft on and off". At 50% request will be active only one at a time - if you have 2 heaters. 
For the rest of request's the on/off ratio will be in between. This is a great algo,
